### PR TITLE
Sync sickness

### PR DESCRIPTION
**Pull request by erbridge**
We set the end date of open sickness to today to get up to date information about absences into Productive even if they're incomplete.

This builds on top of #4. Please review that first.

### CODE REVIEW

File: lib/breathe_client.rb [hunk 1]
```diff
@@ -42,11 +68,79 @@ def events(after:)
         hash[email] = EventCollection.new(events)
       }
     end
-    memo_wise :events
+    memo_wise :absence_events
+
+    def sickness_events(after:)
```
Comment (jdudley1123) [inline lib/breathe_client.rb:73]: It feels like we've got a some repetition now between the different event types – the `x_by_email` method for example. Is there some refactoring that could be done to make it a bit DRYer?

File: lib/breathe_client.rb [hunk 2]
```diff
@@ -56,5 +150,37 @@ def absences(after:)
         .data[:absences]
     end
     memo_wise :absences
+
+    def sicknesses(after:)
+      client
+        .sicknesses
+        .list(start_date: after, exclude_cancelled_sicknesses: true)
```
Comment (jdudley1123) [inline lib/breathe_client.rb:157]: It bugs me that this all on one line, but the equivalent for trainings is on two lines...

Comment (jdudley1123) [inline lib/breathe_client.rb:157]: It bugs me that this all on one line, but the equivalent for trainings is on two lines...

**Review by jdudley1123 — APPROVED** [review]
I would suggest making this a bit DRYer, but I don't think it's a major issue, so I have no problem with this going through as is.

File: lib/breathe_client.rb (hunk 2 above)
Comment (erbridge) [reply lib/breathe_client.rb:157]: Trainings pushed it over 80 characters, but I'll make them consistent (it was also something I thought about at the time).

File: lib/breathe_client.rb (hunk 1 above)
Comment (erbridge) [reply lib/breathe_client.rb:73]: Probably, but I don't have a good way to call out the API difference between sickness / training and absence. Do you?

**Pull request merged**